### PR TITLE
Add test coverage for ComposerInstalledJsonPackagesProvider installed.json happy path

### PR DESCRIPTION
**Pull request by ben-challis**

### CODE REVIEW

File: tests/support/PackageAsserter.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    {
+        Assert::assertContains($license, $this->package->licenses);
+
+        return $this;
+    }
+
+    /**
+     * @param list<string> $licenses
+     */
+    public function hasExactLicense(array $licenses): self
```
Comment (marmichalski) [inline tests/support/PackageAsserter.php:43]:
```suggestion
    public function hasExactLicenses(array $licenses): self
```

?

Comment (ben-challis) [reply tests/support/PackageAsserter.php]: Thanks! https://github.com/Lendable/composer-license-checker/pull/61/commits/b62ddf35092846c70bb8ca8c3f00b8bcf7d32415

File: tests/unit/PackagesProvider/ComposerInstalledJsonPackagesProviderTest.php [hunk 2]
```diff
@@ -27,6 +31,105 @@ protected function tearDown(): void
...
+            ],
+            new Packages([
+                new Package(new PackageName('foo/bar'), []),
+            ]),
+            true,
+        ];
+    }
+
+    /**
+     * @param array<mixed> $data
```
Comment (marmichalski) [inline tests/unit/PackagesProvider/ComposerInstalledJsonPackagesProviderTest.php:121]: ugly

Comment (ben-challis) [reply tests/unit/PackagesProvider/ComposerInstalledJsonPackagesProviderTest.php:121]: https://github.com/Lendable/composer-license-checker/pull/65

**Pull request merged**